### DISCUSSION TITLE
format html element content in a new-line and not a single line

### DISCUSSION THREAD

**Discussion by electriquo** [prettier/prettier#15588] [Q&A]
i was looking at [prettier options](https://prettier.io/docs/en/options.html) but couldn't find how to configure prettier to format html element content in a new line rather than a single line.

for example, from:

```html
<h1>header</h1>
```

to:

```html
<h1>
  header
</h1>
```

how to configure prettier to format html element content in a new-line and not single line?

**Comment by sosukesuzuki** [comment]
We don't add any option for this. Sorry.

**Reply by electriquo** [reply]
no need to be sorry. thank you for prettier and for the reply :)
